docs/quick_start.md

```diff
@@ -3,7 +3,56 @@ In this quick start guide, you'll get up and running with SQLMesh's scaffold gen
 
 Before beginning, ensure that you meet all the [prerequisites](prerequisites.md) for using SQLMesh.
 
-## 1. Create a SQLMesh project
+## Project structure
+
+This project demonstrates key SQLMesh features by walking through the SQLMesh workflow on a simple data pipeline. This section describes the project structure and the SQLMesh concepts you will encounter as you work through it.
+
+The term "pipeline" is generic and usually refers to more than a collection of models and the relationships among them. We use instead the more specific term "directed acyclic graph" (or "DAG") to refer to the models and their relationships in a SQLMesh project. In this context, "graph" is synonymous with "network" - each model is a node in the network, and two models are linked if one of them `SELECT`s from the other. A SQLMesh project's model graph is "acyclic" because it cannot contain cycles where two models both `SELECT` from one another, either directly or indirectly via other models.
```

Comment:
> In this context, "graph" is synonymous with "network"

I'm not sure that by introducing yet another term (network) we help explain what other terms are. 

Can we just get rid of pipelines and networks altogether? I'd rather explain model as a transformation step and then mention that models / transformation depend on each other and, thus, form a graph of dependencies. I think that's all we need:
* Transformation
* Model as expression of transformation
* Graph to capture dependencies between models